src/codecs/decoding/decoder.rs

```diff
@@ -35,9 +38,16 @@ impl Decoder {
         Self {
             framer,
             deserializer,
+            log_namespace: LogNamespace::Legacy,
         }
     }
 
+    /// Sets the path prefix where all deserialized data will be placed.
```

Comment:
Is this comment correct? Log namespaces have nothing to do with path prefixes, right?

Reply:
This was a really old comment from before the RFC when we were going to prefix all data. I fixed the comment.